pkg/test/utils/test_context.go

```diff
@@ -32,8 +32,8 @@ const prometheusOperatorVersion = "0.33"
 const certmanagerURL = "https://github.com/jetstack/cert-manager/releases/download/%s/cert-manager.yaml"
```

Comment:
I'm not sure that we want to move these utils into `pkg`, since that indicates that this is a supported library. Can we leave these in `test/e2e` for now and make the library decision later?